changelog/_unreleased/2025-07-07-use-minimal-search-term-length-in-config-tables.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Use minimal search term length in config tables
+issue: 8018
+---
```

Comment:
also maybe deprecate the parameter and env variable in src/Core/Framework/DependencyInjection/data-abstraction-layer.xml:11